### PR TITLE
feat(card-in-card): v2 updates

### LINKED ISSUE #10384: [Component] Card in card v2 WC
### User story
Update Card in card to use Carbon v11 styles and add new features

### Additional information
See [Figma specs](https://www.figma.com/file/ib2Ecqfvp3NWBiKVBwSHc7/Card-in-card-v2?node-id=71%3A2496&t=8CljL6qEUDL21Ijy-1) for details

**Style changes**
- [ ] border around media
- [ ] border around inset card
- [ ] remove play button overlay
- [ ] depends on card default
- [ ] v11 color tokens
- [ ] v11 type tokens

**Additional features**
Reference v2 Component [review notes](https://ibm.ent.box.com/notes/1196282090517)
- [ ] Opt-in aspect ratios applied to inner card
- [ ] Card in card within a card group should automatically inherit the aspect ratio applied to the card group cards

### Acceptance criteria
_TBD – fill in new acceptance criteria for v2 ?_

### PR DESCRIPTION
**Pull request by ariellalgilmore**
### Related Ticket(s)

Closes #10384

### Description

Card in Card v2 updates
### Changelog

**New**

- new v2 styles
- border around media

### CODE REVIEW

**Review by RichKummer — COMMENTED** [review]
@ariellalgilmore this is looking good and the border seems to be coming in correctly. I noticed a few things that differ from the specs:

1. The spacing below the label should be `4px`, currently it's showing up as `8px`
Screenshot 2023-10-03 at 3 59 13 PM

2. The focus state looks off – currently it looks like the borders are expanding outside the component, instead they should be inside. Also the white inner border seems much larger.

Current build:
Screenshot 2023-10-03 at 3 59 13 PM

Specs:
Screenshot 2023-10-03 at 3 59 35 PM

**Comment by ariellalgilmore** [comment]
Thanks @RichKummer! updated!

**Review by kennylam — APPROVED** [review]
LGTM!

**Review by RichKummer — APPROVED** [review]
LGTM! Thanks for the fixes @ariellalgilmore

File: packages/web-components/src/components/card-in-card/card-in-card.ts [hunk 1]
```diff
@@ -25,23 +25,16 @@ const { prefix, stablePrefix: c4dPrefix } = settings;
...
+    const { ctaType, videoName, videoThumbnailUrl, _hasImage: hasImage } = this;
     const image =
       hasImage || ctaType !== CTA_TYPE.VIDEO
         ? undefined
         : html`
             <c4d-card-in-card-image
               alt="${ifDefined(videoName)}"
-              default-src="${ifDefined(thumbnail || videoThumbnailUrl)}">
-              ${PlayVideo({ slot: 'icon' })}
+              default-src="${videoThumbnailUrl}">
```
Comment (emyarod) [inline packages/web-components/src/components/card-in-card/card-in-card.ts:37]: this just helped me figure out what was going on with carousel! :sob:

**Pull request merged**